rosdep/python.yaml

```diff
@@ -3107,6 +3107,19 @@ python-opengl:
     xenial: [python-opengl]
     yakkety: [python-opengl]
     zesty: [python-opengl]
+python-openhsi:
+  debian:
+    pip:
+      packages: [openhsi]
+  fedora:
+    pip:
+      packages: [openhsi]
+  gentoo:
+    pip:
+      packages: [openhsi]
```

Comment:
According to the contributing guidelines:

> As a reminder pip rules should not be used on Gentoo. See above for the Gentoo specific guidelines.

Reply:
fixed in https://github.com/ros/rosdistro/pull/32533/commits/e929a52b8ee2d29789545cee1a2a5706bac22a0d